### ISSUE TITLE
Along with default configs add other popular architecture

### ISSUE THREAD

**Issue by AadityaMunjal** [AadityaMunjal/scraply#18]
1. Imagenet 
2. YOLO

**Comment by mehek-niwas** [comment]
- rename "default config" --> "default model" 
- include training settings to switch with the model as well 
^^ for example, when selecting the MNIST default model, the training loss setting needs to change from BCE to Cross Entropy as well
